WL-3293 Take 2

buckett opened the pull request.

nicholaswilson100 committed:
WL-3293

Rename PDA 'Log in' button text to 'Oxford Users'

nicholaswilson100 committed:
WL-3293

Change PDA portal 'Log in' button to 'Oxford Account'

Pulling 'Oxford Account' from existing property in sakai.properties

nicholaswilson100 committed:
WL-3293

Change PDA portal 'Log in' button to 'Oxford Account'

Reverting sitenav properties files changes

nicholaswilson100 committed:
WL-3293

Drop the 'pda_login' property as it's unused from the properties files

nicholaswilson100 committed:
WL-3293

Rolling back a single properties file as it's encoded differently - Drop the 'pda_login' property as it's unused from the properties files

nicholaswilson100 commented:
Above I've reverted the sitenav_tr_TR.properties file to its state before I removed the pda_login property.

buckett commented:
Github is still showing loads of changes for that file.

nicholaswilson100 committed:
SAK-19280, Turkish Translation

git-svn-id: https://source.sakaiproject.org/svn/portal/trunk@104263 66ffb92e-73f9-0310-93c1-f5514f145a0a

Conflicts:
	portal-impl/impl/src/bundle/sitenav_tr_TR.properties

Reverting to commit before I touched the file (for WL-3293)

nicholaswilson100 commented:
I think IDEA is being wierd about this so in a terminal I checked out the version of the file before I changed it (git checkout 497d3fb -- portal-impl/impl/src/bundle/sitenav_tr_TR.properties) and committed and pushed that in gitk.

But it still seems to show lots of changes?
